README.md

```diff
@@ -1,26 +1,39 @@
 # Introduction
 
 FreeDV beacon:
+
 + Listens for FreeDV signals, then transmits a reply.
 + Supports FreeDV 1600/700C/700D modes.
 + Places the received signal files on a web server.
 + Requires a Linux machine with a sound card and RS232-PTT (or Hamlib CAT) interface to your radio.
 + Just one sound card is required.
 + Can run on machines as small as a Raspberry Pi.
 
-When a "trigger" string is detected in the rx FreeDV text message (e.g. "hello beacon", or the beacon callsign), the beacon will transmit a signal report back to you.
+When a "trigger" string is detected in the rx FreeDV text message (e.g. "hello beacon", or
+the beacon callsign), the beacon will transmit a signal report back to you.
 
```

Comment:
@Kahn can you pls explain the changes here?  Are you adding newlines?

Reply:
Yup, might be overkill here but in general try and keep markdown line lengths between 80 - 100 characters for readability in an editor. The rendered output remains a single paragraph.

It's ultimately optional though, just a force of habit for me :)

Reply:
Yes I've adopted (are in the process of adopting) the convention of no line breaks, so we better stick with that in this file too ....

Reply:
@Kahn could you pls :point_up: